qutebrowser/mainwindow/tabwidget.py

```diff
@@ -652,7 +652,8 @@ def _draw_icon(self, layouts, opt, p):
         icon_state = (QIcon.On if opt.state & QStyle.State_Selected
                       else QIcon.Off)
         icon = opt.icon.pixmap(opt.iconSize, icon_mode, icon_state)
-        p.drawPixmap(layouts.icon.x(), layouts.icon.y(), icon)
+        center_align_flags = Qt.AlignVCenter | Qt.AlignHCenter
```

Comment:
There's `Qt.AlignCenter` already defined as `Qt.AlignVCenter | Qt.AlignHCenter` :wink:

Reply:
Aaand it's only a few lines lower in the documentation :facepalm: